src/wp-includes/class-wp-term.php

```diff
@@ -99,6 +99,86 @@ final class WP_Term {
 	 */
 	public $filter = 'raw';
 
+	/**
+	 * Stores the object ID.
+	 *
+	 * @since 6.6.0
+	 * @var int|null
+	 */
+	public $object_id;
+
+	/**
```

Comment:
Hey @anton-vlasenko Where are these properties coming from? Are they _actual properties Core is adding_, meaning they are indeed valid dynamic properties that need to added to the class?

Reply:
Yes, I believe they are actual properties that Core is using/interacting with.
As far as I remember, I discovered this when I ran the unit tests, so I had to add these properties to make the unit tests pass.

Reply:
I pushed 884dd17e6568d86336998826df06ab9480823784 to test if the unit tests indeed fail and if these properties are used by Core.

Reply:
So yes, it proved my theory. The unit tests are failing, and the [failure messages](https://github.com/WordPress/wordpress-develop/actions/runs/10409764306/job/28830153132?pr=6426) clearly indicate which properties need to be added to the WP_Term class.